examples/mgo_espresso_d3q/2_run_simulation.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+"""
+MgO kALDo calculation demonstrating NAC correction
+Calculates harmonic properties and thermal conductivity.
+
+Adjust k points in "Replica Settings" under the Configuration header, lower
+will run quicker but less accurately.
+"""
+
+# Edit the number of threads here to control the threading.
+nthr = 32
+import tensorflow as tf
+tf.config.threading.set_intra_op_parallelism_threads(nthr)
+tf.config.threading.set_inter_op_parallelism_threads(nthr)
+# For GPU control, uncomment this code block!
+# gpu_index = 0 # which gpu to select.
+# These indices match with the output from the nvidia-smi command on your cli
+# devices = tf.config.list_physical_devices("GPU")
+# tf.config.set_visible_devices(devices[gpu_index])
+
+import numpy as np
+from kaldo.observables.harmonic_with_q import HarmonicWithQ
+from kaldo.forceconstants import ForceConstants
+from kaldo.phonons import Phonons
+from kaldo.controllers.plotter import plot_dos, plot_dispersion
+from kaldo.conductivity import Conductivity
+
+# ============================================================
+# Configuration
+# ============================================================
+# Dispersion settings
+n_points = 200
+path_string = 'GXUGLW'
+unfold_bool = True
+output_folder = 'plots/'
+
+# Replica settings
+k = 9
+kpt_array = np.array([k, k, k])
+nrep = int(9)
+nrep_third = int(5)
+supercell = np.array([nrep, nrep, nrep])
+third_supercell = np.array([nrep_third, nrep_third, nrep_third])
+
+# Simulation Settings
+temperature = 300
+classical_stats = False
+# True/False means Classical/Quantum heat capacity + phonon populations
+
+# Correction-dependent Settings
+# True = using NAC
+# False = without NAC
+nac_folder_dic = {
```

Comment:
Difficult to read. I'd suggest something in the for loop
`io_folder = 'forces' if is_nac_corrected else 'force_no_charges'`